.speakeasy/gen.yaml

```diff
@@ -16,13 +16,13 @@ generation:
     oAuth2ClientCredentialsEnabled: true
     oAuth2PasswordEnabled: true
 python:
-  version: 0.5.2
+  version: 0.5.3
   additionalDependencies:
     dev: {}
     main:
       PyYAML: ^6.0.2
       pyjwt: ^2.9.0
-      rich: ^13.9.4
+      types-PyYAML: ^6.0.12
```

Comment:
this is needed as a `dev` dependency only I would think.

Reply:
yeah, looks like that. will add it there.